### PR TITLE
Update ref to the RATS architecture RFC

### PR DESCRIPTION
**Pull request by thomas-fossati**
Signed-off-by: Thomas Fossati <thomas.fossati@arm.com>

### CODE REVIEW

File: draft-tschofenig-rats-psa-token.md [hunk 1]
```diff
@@ -426,14 +427,14 @@ Each entry in the Software Components list describes one software component
...
-will typically see the result of the verification process from the Verifier in
-form of an attestation result, rather than the "naked" PSA token from the
-attesting endpoint.  Therefore, a relying party is not expected to understand
-the Software Components claim.  Instead, it is for the Verifier to check this
-claim against the available endorsements and provide an answer in form of an
-"high level" attestation result, which may or may not include the original
-Software Components claim.
+Note that, as described in {{?RFC9334}}, a relying party will typically see the
+result of the verification process from the Verifier in form of an attestation
+result, rather than the "naked" PSA token from the attesting endpoint.
```
Comment (adrianlshaw) [inline draft-tschofenig-rats-psa-token.md:432]: Is the word naked really needed here?

**Review by adrianlshaw — APPROVED** [review]
LGTM

File: draft-tschofenig-rats-psa-token.md (hunk 1 above)
Comment (thomas-fossati) [reply draft-tschofenig-rats-psa-token.md:432]: even better :-)

Comment (thomas-fossati) [inline draft-tschofenig-rats-psa-token.md:432]:
```suggestion
result, rather than the PSA token from the attesting endpoint.
```

Comment (thomas-fossati) [reply draft-tschofenig-rats-psa-token.md]: removed in [f042562](https://github.com/thomas-fossati/draft-psa-token/pull/73/commits/f042562bc7a8a1ade7a242015578ada86d6611c4)

**Pull request merged**